src/routes/ad.ts

```diff
@@ -1,11 +1,13 @@
 import * as Router from 'koa-router';
 
-import { getAds, getLiquidity, getSavedSearchAds } from '../controllers/adController';
+
+import { getAds, getLiquidity, getMostLiquid,getSavedSearchAds } from '../controllers/adController';
 
 const router = new Router();
 
 router.post('/', getAds);
 router.get('/saved', getSavedSearchAds);
+router.get('/most_liquid', getMostLiquid);
 router.post('/get-liquidity', getLiquidity);
```

Comment:
Please don't use minus sign in urls. This is a moveton. Use lodashes insted as you did above. Please refactor.